website/common/locales/en/groups.json

```diff
@@ -161,6 +161,7 @@
   "abuseFlagModalBody": "Are you sure you want to report this post? You should <strong>only</strong> report a post that violates the <%= firstLinkStart %>Community Guidelines<%= linkEnd %> and/or <%= secondLinkStart %>Terms of Service<%= linkEnd %>. Inappropriately reporting a post is a violation of the Community Guidelines and may give you an infraction.",
   "abuseFlagModalButton": "Report Violation",
   "abuseReported": "Thank you for reporting this violation. The moderators have been notified.",
+  "pmReported": "Thank you for reporting this message. The moderators have been notified. You can now delete the message if you wish.",
```

Comment:
`pmReported` is used for the notification text, do you mean this or the text under a private message?

Reply:
Yes, I mean the notification text. It seems to be shown to the user as soon as they've submitted the form, so at that point we can't be sure that the moderators have received the slack post and email, and so we don't want the user to prematurely delete the message they tried to report.

The text under the private message doesn't need to have the deletion tip removed since it doesn't have the same potential problem (it's only shown to the user after the code has confirmed that the email and slack message were sent correctly, right?)

Reply:
yes the text under the private message is only show (if the message has flags OR when the API-Result comes back with an `ok` )